src/Symfony/Component/Notifier/Bridge/Discord/DiscordTransport.php

```diff
@@ -66,7 +66,7 @@ protected function doSend(MessageInterface $message): SentMessage
         $content = $message->getSubject();
 
         if (\strlen($content) > 2000) {
-            throw new LogicException(sprintf('The subject length of "%s" transport must be less than 2000 characters.', __CLASS__));
+            throw new LogicException('The subject length of a Discord message must not exceed 2000 characters.');
```

Comment:
I am wondering if this should be an InvalidArgumentException 🧐

Reply:
[LengthException](https://www.php.net/manual/fr/class.lengthexception.php) could be a good fit. 
That's for 5.x anyway I suppose :)

Reply:
PR welcome now that it's merged.